DATA/UNIVERSE/SYSTEMS/Bw10/Bw10.ini

```diff
@@ -592,7 +592,7 @@ faction = co_me_grp, 0.3 ; Deep Space Engineering
 
 [Object]
 nickname = Bw10_Trade_Lane_Ring_8
-ids_name = 260893
+ids_name = 260894

```

Comment:
Why did all IDs change again?

Reply:
I change it to the most recent file on github. Not sure what to tell you.